cogs/utils/botdata.py

```diff
@@ -121,6 +121,13 @@ def __init__(self, botdata, discord):
 			"type": types.Boolean,
 			"description": "If enabled, mango will private message you when a new dota patch gets released",
 			"example": "enable"
+		},
+                {
+                        "key": "dmdotablog",
+                        "default": None,
+                        "type": types.Boolean,
+                        "description": "Enabling this will let mangobyte dm you about Dota blog updates",
+                        "example": "enable"
```

Comment:
I think some/all of these are using spaces for indentation instead of tabs? Usually this would be a super minor thing, but I've known python to get weird about this and break sometimes if theres inconsistent indentation in files.

Reply:
Ah, my bad. I didn't check the tab behavior I had set on the editor.  I will go through and fix this, and also check the code from the last pull request, as it probably has the same problem.